versions/3.0.4.md

```diff
@@ -280,7 +277,7 @@ The metadata MAY be used by the clients if needed, and MAY be presented in editi
 | <a name="info-terms-of-service"></a>termsOfService | `string` | A URL for the Terms of Service for the API. This MUST be in the form of a URL. |
 | <a name="info-contact"></a>contact | [Contact Object](#contact-object) | The contact information for the exposed API. |
 | <a name="info-license"></a>license | [License Object](#license-object) | The license information for the exposed API. |
-| <a name="info-version"></a>version | `string` | **REQUIRED**. The version of the OpenAPI document (which is distinct from the [OpenAPI Specification version](#oas-version) or the version of the API being described). |
+| <a name="info-version"></a>version | `string` | **REQUIRED**. The version of the OpenAPI Description (which is distinct from the [OpenAPI Specification version](#oas-version) or the version of the API being described). |
```

Comment:
We have debated whether this is the version of the OAD or the version of the document, and I thought we decided on the document.

Consider a multi-document OAD that uses a shared `components`-only "syntactically complete" document.  That document is maintained separately from any of the entry documents that reference it, so it needs its own `version`.

I think the `version` field of the entry document is suitable for use as the overall version of the OAD, although I don't think we discussed that.

Reply:
I don't want to go to different version numbers for different parts of an OAD and think Mike's change is fine as we don't define "OpenAPI document".

Reply:
>I think the version field of the entry document is suitable for use as the overall version of the OAD, although I don't think we discussed that.
>
AFAIK, we have not.  I agree with this statement, but mindful of @ralfhandl comment about different versions spread across different OADs.  Maybe  we should include additional semantics whereby the version of the root OAD  is the overall version of the OAD when you do not specify a version on a shared component OAD.

Reply:
To me the comments from @miqui and @darrelmiller seem to say the same:
* The version field of the entry document is the one and only version of a multi-document OpenAPI Description.
* If non-entry documents are reached via reference from a 2.0 or 3.0 entry document, it does not matter whether they have a version field; it is never interpreted and references are resolved on the in-memory representation of a "JavaScript" model resulting from plain JSON/YAML parsing of the text documents.
* If the entry document is 3.1, things apparently are different and I leave the parsing to @handrews 😎

Reply:
I think this interpretation is a valid interpretation for 3.0 OADs. I think there may be other interpretations, including ones that are more consistent with 3.1. So we should let this be "implementation defined" in 3.0 with enough explanation to allow any of these valid interpretations while ruling out any others.

Reply:
Reverted this change but added a clarification that the version is also not the version of the OAD, since as @darrelmiller pointed out the "version" of the OAD could change when some referenced component outside this document changes.